<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharEntryDialog</class>
 <widget class="QDialog" name="CharEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Preview</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="label_Char">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>132</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_CharRelated">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>132</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Character Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_Vowel">
        <property name="text">
         <string>Vowel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_DependantVowel">
        <property name="text">
         <string>Dependant Vowel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_Consonant">
        <property name="text">
         <string>Consonant</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButton_Digit">
        <property name="text">
         <string>Digit</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="radioButton_SpecialSymbol">
        <property name="text">
         <string>Special Symbol</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="radioButton_Virama">
        <property name="text">
         <string>Virama / Halant</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="radioButton_None">
        <property name="text">
         <string>None</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_Vowel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_DepRefVowel">
           <property name="toolTip">
            <string>The corresponding dependant/independant form of the character (in Hex)</string>
           </property>
           <property name="inputMask">
            <string>hhhh; </string>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Phonetic Strings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_PhoneticString">
        <property name="toolTip">
         <string>Enter Phonetic String here</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_Add">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_Remove">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_Ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CharEntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CharEntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_DependantVowel</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_DepRefVowel</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Vowel</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_DepRefVowel</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Ok</sender>
   <signal>clicked()</signal>
   <receiver>CharEntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
